database/migrations/000060_rule_instances.up.sql

```diff
@@ -0,0 +1,39 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS rule_instances(
+    id           UUID NOT NULL DEFAULT gen_random_uuid() PRIMARY KEY,
+    profile_id   UUID NOT NULL REFERENCES profiles(id) ON DELETE CASCADE,
+    rule_type_id UUID NOT NULL REFERENCES rule_type(id),
+    name         TEXT NOT NULL,
```

Comment:
It is case sensitive in the existing code, and so I preserved that behaviour. I think it will make the migration harder if we introduce that change up front.